node/pkg/watchers/evm/connectors/poller.go

```diff
@@ -171,9 +171,9 @@ func (b *BlockPollConnector) getBlock(ctx context.Context, logger *zap.Logger, n
 	}
 
 	type Marshaller struct {
-		Number     *ethHexUtils.Big
-		Hash       ethCommon.Hash `json:"hash"`
-		Difficulty *ethHexUtils.Big
+		Number        *ethHexUtils.Big
+		Hash          ethCommon.Hash `json:"hash"`
+		L1BlockNumber *ethHexUtils.Big
```

Comment:
Let's add a comment saying this field is only populated in Arbitrum

Reply:
Comment added.

Reply:
Please make it a godoc string such that it shows up in IDE autocompletion.

E.g.
```
// L1BlockNumber is the L1 block number an Arbitrum batch containing this block was submitted. This field is only populated when connecting to Arbitrum
L1BlockNumber *ethHexUtils.Big
```

Reply:
Done.